<ui version="4.0" >
 <class>CharacterHighlighting</class>
 <widget class="QWidget" name="CharacterHighlighting" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>297</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>Underlining:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="buddy" >
      <cstring>underlineStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="KComboBox" name="underlineStyle" />
   </item>
   <item row="1" column="1" >
    <widget class="KComboBox" name="underlineLineStyle" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="KColorButton" name="underlineColor" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Capitalization</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Normal</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="normal" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Uppercase</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QRadioButton" name="uppercase" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Lowercase</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QRadioButton" name="lowercase" />
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Small Caps</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QRadioButton" name="smallcaps" />
      </item>
      <item rowspan="3" row="0" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1" >
       <widget class="QRadioButton" name="capitalize" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Capitalize</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="textLabel2" >
     <property name="text" >
      <string>Strikethrough:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QComboBox" name="strikethroughStyle" />
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="strikethroughLineStyle" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="KColorButton" name="strikethroughColor" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
  <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kcolorbutton.h</include>
  <include location="local" >kcombobox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
